src/oas2/schema.ts

```diff
@@ -0,0 +1,22 @@
+import { isObject } from 'lodash';
+import { Schema } from 'swagger-schema-official';
+
+export function getExamplesFromSchema(data?: Schema) {
+  let examples;
+
+  if (isObject(data)) {
+    if ('x-examples' in data) {
+      examples = data['x-examples'];
```

Comment:
It seems like there is a strong assumption that the data in the `x-examples` property is correct. We should validate this

Reply:
Right, I've added check if it's an object. That should be enough in that case.